src/apis/doctor/doctor.controller.js

```diff
@@ -559,4 +559,11 @@ router.get('/doctors/:doctorId/history', function(req, res) {
     });
 });
 
+router.post('/doctors/:doctorId/genpdf', function(req, res) {
```

Comment:
Change `/doctors/:doctorId/genpdf` to `/doctors/:doctorId/prescriptions`

Reply:
Or if it's only related to creation of pdf files, then go with `/doctors/:doctorId/files/pdf`

Reply:
But the name doesn't imply if we are downloading a file or generating a new one